### DISCUSSION TITLE
Inject Minecraft Bedrock

### DISCUSSION THREAD

**Discussion by cngege** [ocornut/imgui#5129] [New Users Build/Link/Run issues ONLY!]
为我将ImGui写在（C++）DLL里面，然后注入到Minecraft Bedrock,为什么不显示UI？
Write ImGui in a (C++) DLL for me and inject it into Minecraft Bedrock, why doesn't the UI show up?

我注入到记事本是成功的，可以显示ImGui
I inject into notepad is successful, can display ImGui

#pragma once
#include "imgui\imconfig.h"
#include "imgui\imgui.h"
#include "imgui\imgui_impl_dx11.h"
#include "imgui\imgui_impl_win32.h"
#include "imgui\imgui_internal.h"
#include "imgui\imstb_rectpack.h"
#include "imgui\imstb_textedit.h"
#include "imgui\imstb_truetype.h"

**Comment by ocornut** [comment]
Sorry but #1586

**Comment by iWanheda** [comment]
bedrock edition uses dx12 not dx11, unless you're creating some kind of overlay or whatever, but i won't give more info since @ocornut said #1586
